modules/ROOT/pages/log-search-query-syntax.adoc

```diff
@@ -63,7 +63,11 @@ Boolean operators must be ALL CAPS.
 or
 
 `"monitoring anypoint" OR monitoring`
-|`NOT`| The `NOT` operator excludes documents that contain the term after `NOT`. This is equivalent to a difference using sets. The symbol `!` can be used in place of the word `NOT`.|
+|`NOT`| The `NOT` operator excludes documents that contain the term after `NOT`. This is equivalent to a difference using sets. The symbol `!` can be used in place of the word `NOT`.|To search for documents that contain "anypoint monitoring" but not "results", use the query:
+`"anypoint monitoring" NOT "results"` +
+[NOTE]
+The `NOT` operator can't be used with just one term. For example, the following search returns no results: +
+`NOT "results"`
```

Comment:
Could we use a word different from results? I'm concerned that people might be confused about the difference between getting no results and getting no results that contain the word "results."